addons/bestja_frontend_ucw/static/src/css/fonts/OpenSans-CondBold-webfont.svg

```diff
@@ -0,0 +1,634 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
I think that we’ve discussed that CDN-hosted fonts will be ultimately better, so those should be removed from the repository.